src/main/resources/XWiki/SuggestDisplay.xml

```diff
@@ -453,6 +516,31 @@ ul.picker-list .picker-item {
     #end
     ## now display the item in edit mode, to have the suggest and all.
     $doc.displayEdit($propertyClass, $prefix, $object)
+
+    ## Save the className and the class property name in hidden picker (In editItemInPopups mode)
+    #if("$!SuggestDisplay_itemsEditable" != "" &amp;&amp; $SuggestDisplay_itemsEditable == true)
+       ## New js/css calls
+       #set($discard = $xwiki.ssx.use("XWiki.SuggestDisplayEditItems"))
+       #set($discard = $xwiki.jsx.use("XWiki.SuggestDisplayEditItems"))
+       #set ($attachmentPickerDocName = 'XWiki.AttachmentSelector')
+       $xwiki.ssx.use($attachmentPickerDocName)
+       $xwiki.jsx.use($attachmentPickerDocName)
+       ## Wysiwyg
+       #set($lazy = false)
+       #wysiwyg_import($lazy)
+       ## Date picker : TODO
+       &lt;input type="hidden" id="${prefix}_className" value="$object.getxWikiClass().getName()"/&gt;
+       &lt;input type="hidden" id="${prefix}_propertyName" value="$name"/&gt;
+       &lt;input type="hidden" id="itemParent" value="$doc.getDocumentReference()"/&gt;
+       #if("$!SuggestDisplay_editModeSpace" != "")
+       &lt;input type="hidden" id="itemSpace" value="$!SuggestDisplay_editModeSpace"/&gt;
+       #else
+       &lt;input type="hidden" id="itemSpace" value="$doc.getSpace()"/&gt;
+       #end
+       &lt;input type="hidden" class="inEditItemsMode" value="1"/&gt;
+       &lt;input type="hidden" class="currentWiki" value="$context.getDatabase()"/&gt;
```

Comment:
Please use `$services.wiki.currentWikiId` instead of `$context.getDatabase()` (which should be deprecated).